app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java

```diff
@@ -991,6 +991,16 @@ private String checkYoutubeOverride(String aUri) {
             String result = aUri;
             result += aUri.contains("?") ? "&" : "?";
             result += "disable_polymer=1";
```

Comment:
should we check `uri.getQueryParameter("disable_polymer")` to be consistent with these below?

Reply:
Done

Reply:
thank you